include/setup.php

```diff
@@ -317,9 +317,19 @@
 // Loads English localized strings by default (must go before config.php)
 require 'languages/english.php';
 
+// Try to support one WebSVN-installation hosting multiple different SVNParentPaths, distinguished
+// by their location. Per location the web server needs to set some environment variable providing
+// the path to the config to include.
+//
+// https://stackoverflow.com/questions/3050444/when-setting-environment-variables-in-apache-rewriterule-directives-what-causes
+$envConfPath = preg_grep('/^(?:REDIRECT_)*WEBSVN_PATH_CONF$/', array_keys($_SERVER));
+$envConfPath = $_SERVER[array_values($envConfPath)[0]];
+
 // Get the user's personalised config (requires the locwebsvnhttp stuff above)
 if (file_exists('include/config.php')) {
 	require_once 'include/config.php';
+} elseif (!empty($envConfPath)) {
```

Comment:
Should a specific file override the default one?

Reply:
I'm not overriding currently, but replacing, because that's the only concept WebSVN implemented: Use one `config.php`, I only changed it's path. But if layering is of interest then yes, more specific files should override settings in the default or more general config.

With additionally including the file set by the environment variable one would already have that sort of layering. But the name of the variable doesn't reflect that currently and I wasn't sure if layering should only be supported by using a setup like mine. Being able to alternatively configure WebSVN using a setup like mine instead made sense to me.